### ISSUE TITLE
[Enhancement] Startup on Login

### ISSUE THREAD

**Issue by brianx87** [jshackles/Steam_Invite_Spam_Filter#2]
Can you add the option to startup on windows login please.

**Comment by Scrxtchy** [comment]
open shell:startup, and create a shortcut to the application in that folder

**Comment by ghost** [comment]
Yes, just copy the executable to `C:\Users\user\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup`.

**EDIT**: Except doesn't start for me...
